test/setup.spec.js

```diff
@@ -382,45 +382,86 @@ describe('setup', () => {
       .then(done);
   });
 
+  it('should unwrap on the template', () => {
+    const vm = new Vue({
+      setup() {
+        const r = ref('r');
+        const nested = {
+          a: ref('a'),
+        };
```

Comment:
Maybe we could cover deep nested as well.

Reply:
Added a few more, can you check that's good?